docs/use_bls_wallet_clients.md

```diff
@@ -39,55 +37,91 @@ If you would like to deploy to a remote network, see [Remote development](./remo
 ## Send a transaction
 
 ```typescript
-import { readFile } from "fs/promises";
+import { readFile } from 'fs/promises';
+
+// import fetch from 'node-fetch'; // Add this if using nodejs<18
+import { ethers, providers } from 'ethers';
+import { Aggregator, BlsWalletWrapper, getConfig } from 'bls-wallet-clients';
 
+// globalThis.fetch = fetch; // Add this if using nodejs<18
 
 // Instantiate a provider via browser extension, such as Metamask 
-const provider = providers.Web3Provider(window.ethereum);
+// const provider = new providers.Web3Provider(window.ethereum);
 // Or via RPC
-const provider = providers.JsonRpcProvider();
+const provider = new providers.JsonRpcProvider('https://goerli-rollup.arbitrum.io/rpc');
 // See https://docs.ethers.io/v5/getting-started/ for more options
 
 // Get the deployed contract addresses for the network.
 // Here, we will get the Arbitrum testnet.
 // See local_development.md for deploying locally and
 // remote_development.md for deploying to a remote network.
-const netCfg = await getConfig("../contracts/networks/arbitrum-testnet.json", async (path) => readFile(path));
+const netCfg = await getConfig(
+  '../contracts/networks/arbitrum-goerli.json',
+  async (path) => readFile(path),
+);
 
-const privateKey = "0x...";
+// 32 random bytes
+const privateKey = '0x0001020304050607080910111213141516171819202122232425262728293031';
 
 // Note that if a wallet doesn't yet exist, it will be
 // lazily created on the first transaction.
-const wallet = await BlsWallerWrapper.connect(
+const wallet = await BlsWalletWrapper.connect(
   privateKey,
-  netCfg.contracts.verificationGateway,
+  netCfg.addresses.verificationGateway,
   provider
 );
 
-const erc20Address = "0x...";
+const erc20Address = netCfg.addresses.testToken; // Or some other ERC20 token
 const erc20Abi = [
-    "function transfer(address to, uint amount) returns (bool)",
+    'function mint(address to, uint amount) returns (bool)',
```

Comment:
Although we include `mint` on our test token, and `_mint` is a function in the OpenZeppelin ERC20 implementation it is not part of the standard ERC20 interface. I would recommend keeping this as the more common `transfer` here and below.

```suggestion
    'function transfer(address to, uint amount) returns (bool)',
```

Reply:
I see what you mean, but the good thing about `mint` is it allows this to work without any other set up (sending tokens to the contract wallet some other way first).

What do you think we should do instead?

Reply:
I originally wrote this with the assumption that it could be any arbitrary token. I think if we the example to use our config for populating the contract we could include both and just leave a comment on the `mint`.

```typescript
const erc20Abi = [
    'function transfer(address to, uint amount) returns (bool)',
    // Included on BLS Wallet test token for convenience, remove if using another token.
    'function mint(address to, uint amount) returns (bool)',
];
```